src/planning.py

```diff
@@ -325,11 +325,10 @@ def _skeleton_generator(
                                                        node.atoms):
                 child_atoms = utils.apply_operator(nsrt, set(node.atoms))
                 child_skeleton = node.skeleton + [nsrt]
-                if abstract_policy is not None:
```

Comment:
i don't get why this had to be in this `if`, going to try removing that...

Reply:
we were erring on the side of caution in terms of impacting existing code. I agree it shouldn't matter in principle